modules/javafx.graphics/src/main/native-glass/mac/GlassApplication.m

```diff
@@ -1299,5 +1267,6 @@ + (BOOL)syncRenderingDisabled {
 JNIEXPORT jobject JNICALL Java_com_sun_glass_ui_mac_MacApplication_getPlatformPreferences
 (JNIEnv *env, jobject self)
 {
-    return [PlatformSupport collectPreferences];
+    GlassApplication* app = (GlassApplication*)[NSApp delegate];
+    return [app getPlatformPreferences];
```

Comment:
I think this is the cause of the crash. You cannot assume that `[NSApp delegate]` is a `GlassApplication`. In the case where AWT initializes the toolkit first, it won't be.